client/src/components/DataFiles/DataFilesModals/DataFilesPublicationDownloadModal.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+import { Button, DescriptionList, LoadingSpinner, Message } from '_common';
+import { Modal, ModalHeader, ModalBody } from 'reactstrap';
+import styles from './DataFilesPublicationRequestModal.module.scss';
+import { formatDate, formatDateTime } from 'utils/timeFormat';
+import { useFileDetail, useModal } from 'hooks/datafiles';
+
+export function toBytes(bytes) {
+  if (bytes === 0) return '0 bytes';
+  if (!bytes) return '-';
+  const units = ['bytes', 'kB', 'MB', 'GB', 'TB', 'PB'];
+  const orderOfMagnitude = Math.floor(Math.log(bytes) / Math.log(1024));
+  const precision = orderOfMagnitude === 0 ? 0 : 1;
+  const bytesInUnits = bytes / Math.pow(1024, orderOfMagnitude);
+  return `${bytesInUnits.toFixed(precision)} ${units[orderOfMagnitude]}`;
+}
+
+const DataFilesPublicationDownloadModal = () => {
+  const dispatch = useDispatch();
+  const { toggle: toggleModal, getStatus, getProps } = useModal();
+  const { projectId, rootSystem } = getProps('publicationDownload') || {};
+  const archiveProjectId = (projectId ?? '').split('published.')[1];
```

Comment:
I think this might be better as with some of the test publication systems the naming scheme is a little different

```suggestion
  const archiveProjectId = (projectId ?? '').split('.').pop();
```

Reply:
Yeah it does look like a bug that I can log. That `projectId` is used in other components as well and changing that might break other functionality so I'll add this as a future fix